src/navigation/index.tsx

```diff
@@ -1,11 +1,50 @@
 import React from 'react';
-import { createBrowserRouter } from 'react-router-dom';
+import { Route, Routes } from 'react-router-dom';
 
-import Home from '../pages/Home';
+// pages
+import Account from '../pages/Account';
+import Transfer from '../pages/Transfer';
+import History from '../pages/History';
+import NotFound from '../pages/NotFound';
+import Apps from '../pages/Apps';
+import { usePrivy } from '@privy-io/react-auth';
```

Comment:
Should this be moved before / after pages as it's a library?

Reply:
yes, actually I will go through each import and break in commented sections